tools/ci_build/github/azure-pipelines/templates/py-packaging-stage.yml

```diff
@@ -181,7 +181,7 @@ stages:
               --volume $HOME/.onnx:/home/onnxruntimedev/.onnx \
               -e NIGHTLY_BUILD \
               -e BUILD_BUILDNUMBER \
-              onnxruntimeregistry.azurecr.io/internal/azureml/onnxruntimecpubuild:ch9j \
+              onnxruntimeregistry.azurecr.io/internal/azureml/onnxruntimecpubuild:chan \
```

Comment:
just curious, how do these tag names get chosen/what do they mean?

Reply:
These tag names were auto generated.  9j  and an are build ids.

Reply:
onnxruntimecpubuild and onnxruntimegpubuild are images for manylinux python pipeline.
onnxruntimecentoscpubuild and onnxruntimecentosgpubuild are images for non-python packaging pipelines.

Reply:
i see. is it easy to correlate the tag name with the commit from which it was created? or should we consider using the commit as the tag? just in case we need to rebuild the image for some reason

Reply:
Yes. Please see: https://github.com/MicrosoftDocs/azure-docs/blob/master/articles/container-registry/container-registry-tasks-reference-yaml.md#run-variables